packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -61,9 +62,26 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     private _pointerTravelDistance = 0;
     private _processSelectionOnUp = false;
     private _visibleRegionContainer: Container;
+    private _currLeft: number = 0;
+    private _currTop: number = 0;
     public get visibleRegionContainer() {
         return this._visibleRegionContainer;
     }
+    public get controlOffset() {

```

Comment:
I guess you can remove all the public getter you added as they are not used anymore :-) and this will be good to go

Reply:
Sorry for the delay! Pushed those changes before editing all my code because I needed to move into another branch for some issues yesterday, but should be all good to go now :)